snibbetracker/src/main.c

```diff
@@ -4588,9 +4594,13 @@ static int get_buffer_size_from_index(int i) {
 
 }
 
-static void copy_project_win(const char *name) {
+static void copy_project_win_lin(const char *name) {
     char *read_path = cAllocatorAlloc((1024 * sizeof(char*)), "win path 1");
+    #if defined(platform_windows)
     snprintf(read_path, 1023, "%s%s", "demos\\", name);
+    #else
+    snprintf(read_path, 1023, "%s/../share/lundstroem/snibbetracker/demos/%s", base_dir, name);
```

Comment:
is base_dir NULL here?

Reply:
I obtain the base directory from SDL in the main function.
If this fails with NULL for any reason, I exit with the code 1.
Therefore, NULL is impossible.

Reply:
Ah that is correct, was a little difficult following the execution order in the diff :)